hw/hw3/forms.py

```diff
@@ -0,0 +1,11 @@
+from flask_wtf import FlaskForm
+from wtforms import StringField, PasswordField
+from wtforms.validators import DataRequired, Length, EqualTo, Email
+
+
+class RegistrationForm(FlaskForm):
+    name = StringField('Имя', validators=[DataRequired()])
+    surname = StringField('Фамилия', validators=[DataRequired()])
+    email = StringField('Email', validators=[DataRequired(), Email()])
```

Comment:
Тут можно использовать `EmailField`
```suggestion
    email = EmailField('Email', validators=[DataRequired(), Email()])
```